controllers/image_generator.go

```diff
@@ -640,6 +647,26 @@ func (r *ImageGenerator) createImageConfigMapFromFile() error {
 		return err
 	}
 
+	// For Azure provider set the IMAGE_GALLERY_NAME if its empty
```

Comment:
right..

Reply:
So if the image CM exist already (637) shouldn't we make sure gallery image name is set? IIUC it will fail on gallery creation? is it ok to assume if user create he's own CM he's know he should set it

Reply:
If user creates the CM, then user should set the gallery name. We'll update in the README or instructions which provides steps on manual creation of the image creation job (outside of the operator)